app/services/item_service.rb

```diff
@@ -0,0 +1,15 @@
+class ItemService
+  def self.all_items
+    get_data('/api/v1/items')
+  end
+
+  private
+  def self.conn
+    Faraday.new(url: "http://localhost:3000")
+  end
+
+  def self.get_data(url)
+    response = conn.get(url)
+    JSON.parse(response.body, symbolize_names: true)[:data]
+  end
```

Comment:
These methods are the same in this class and in `MerchantService`. I might consider putting these into something like a `RailsEngineService` which `MerchantService` and `ItemService` could inherit from, to DRY up this code